src/Tests/guitar-adnote.xmz

```diff
@@ -134,6 +135,7 @@ version-revision="6" ZynAddSubFX-author="Nasca Octavian Paul">
 <par name="randomness_amplitude" value="0" />
 <par name="randomness_frequency" value="0" />
 <par_real name="delay" value="0.598425" exact_value="0x3F193265" />
+<par_real name="fadein" value="0" exact_value="0x00000000" />
```

Comment:
Maybe, after you have solved the other comments, you will need to add `fadeout` here aswell?

Reply:
ok